apps/web/vibes/soul/sections/featured-blog-post-list/index.tsx

```diff
@@ -1,39 +1,48 @@
-import { Streamable } from '@/vibes/soul/lib/streamable';
+import { Stream, Streamable } from '@/vibes/soul/lib/streamable';
 import { BlogPost } from '@/vibes/soul/primitives/blog-post-card';
-import { ButtonLink } from '@/vibes/soul/primitives/button-link';
+import { Breadcrumb, Breadcrumbs, BreadcrumbsSkeleton } from '@/vibes/soul/primitives/breadcrumbs';
+import { CursorPagination, CursorPaginationInfo } from '@/vibes/soul/primitives/cursor-pagination';
 import { BlogPostList } from '@/vibes/soul/sections/blog-post-list';
 
-interface Link {
-  label: string;
-  href: string;
-}
-
 interface Props {
   title: string;
   description?: string;
-  cta?: Link;
   posts: Streamable<BlogPost[]>;
+  paginationInfo?: Streamable<CursorPaginationInfo>;
+  breadcrumbs?: Streamable<Breadcrumb[]>;
 }
 
-export function FeaturedBlogPostList({ title, description, cta, posts }: Props) {
+export function FeaturedBlogPostList({
+  title,
+  description,
+  posts,
+  paginationInfo,
+  breadcrumbs: streamableBreadcrumbs,
+}: Props) {
   return (
     <section className="@container">
       <div className="mx-auto max-w-screen-2xl px-4 py-10 @xl:px-6 @xl:py-14 @4xl:px-8 @4xl:py-20">
-        <h1 className="mb-3 font-heading text-4xl font-medium leading-none text-foreground @xl:text-5xl @4xl:text-6xl">
-          {title}
-        </h1>
-
-        {description != null && description !== '' && (
-          <p className="max-w-lg text-lg text-contrast-500">{description}</p>
+        {streamableBreadcrumbs && (
+          <Stream fallback={<BreadcrumbsSkeleton />} value={streamableBreadcrumbs}>
+            {(breadcrumbs) =>
+              Boolean(breadcrumbs.length) && <Breadcrumbs breadcrumbs={breadcrumbs} />
```

Comment:
The breadcrumbs already handle a length of zero so we shouldn't have to check that here

Reply:
Ahh good point